python/ml4ir/base/features/feature_config.py

```diff
@@ -283,6 +283,38 @@ def create_dummy_protobuf(self, num_records=1, required_only=False):
         """
         raise NotImplementedError
 
+    def get_wandb_config(self):
```

Comment:
Would probably be better to call it get_weight_bias_config() or create_hyper_parameters().
Reading the function on its own I didn't get what wandb meant.

Reply:
Sure. I just went with `wandb` because that's their package name. But makes sense to do that for more readability.

Reply:
Renamed to `get_hyperparameter_dict` to be more generic as we might move away from wandb.